src/main/resources/db/changelog/changes/11-create-cart-items-table.yaml

```
@@ -0,0 +1,44 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-cart-items-table
+      author: suprun
+      changes:
+        - createTable:
+            tableName: cart_items
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: shopping_cart_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_cart_items_shopping_cart_id
+                    references: shopping_carts(id)
+              - column:
+                  name: book_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_cart_items_bool_id
+                    references: books(id)
+              - column:
+                  name: quantity
+                  type: INTEGER
+                  constraints:
+                    nullable: false
+                    default: 0
+              - column:
+                  name: is_deleted
+                  type: BOOLEAN
+                  defaultValue: false
+                  constraints:
+                    nullable: false
+      rollback:
+        - dropTable:
+            tableName: cart_items
```

each file should end with exactly one empty line
```suggestion
            tableName: cart_items

```